Runtime/Game/Resources/LootLockerConfig.cs

```diff
@@ -78,7 +78,7 @@ static void CreateConfigFile()
             }
         }
 #endif
-        public static bool CreateNewSettings(string apiKey, string gameVersion, bool onDevelopmentMode, string domainKey, platformType platform = platformType.Unused, DebugLevel debugLevel = DebugLevel.All, bool allowTokenRefresh = false)
+        public static bool CreateNewSettings(string apiKey, string gameVersion, string domainKey, bool onDevelopmentMode = false, platformType platform = platformType.Unused, DebugLevel debugLevel = DebugLevel.All, bool allowTokenRefresh = false)
```

Comment:
Won't flipping these break for current users?

Why not make a new overload?

Reply:
That's the problem with everything being static. This is used in the Init method in LootLockerSDKManager.cs and shouldn't be used by our users. But I mean... They could... I think that's on them to fix though.

Reply:
Fair enough, I just wanted to double check that it's an internal method.

I hate that everything is static 😣